Spanner/src/SpannerClient.php

```diff
@@ -73,6 +73,26 @@
  * $spanner = new SpannerClient();
  * ```
  *
+ * ```
+ * use Google\Cloud\Spanner\SpannerClient;
+ * use Google\Cloud\Spanner\V1\DirectedReadOptions\ReplicaSelection\Type as ReplicaType;
+ *
+ * $spanner = new SpanneClient();
```

Comment:
Also add a snippet test for this?
```suggestion
 * $spanner = new SpannerClient();
```